### ISSUE TITLE
Device Import issue - and others.

### ISSUE THREAD

**Issue by jkw117** [Cacti/plugin_mactrack#84]
I have been trying to get mactrack to work for a little bit now. 
And I keep running into issues where after filling out a form it sits, I see the line go across and see it processing. Then leaves me on the form, if I click on the ie "Devices" icon on the right it'll then refresh the content area with the items I changed. It does this for almost everything. 
Some things are even weirder. I'm in the devices section go to import, and it puts a line on the content screen nothing else... If I refresh the page (F5) sometimes it'll populate it with the form to import..

**Comment by netniV** [comment]
Can you use a screen to gif capture program to document in separate images the various issues?

**Comment by jkw117** [comment]
[cacti-screen-issue.docx](https://github.com/Cacti/plugin_mactrack/files/2257851/cacti-screen-issue.docx)
I've attached a image of what happends on almost each form page. for Ie in this one I just click the save and then I get a form page with a line on it.

**Comment by jkw117** [comment]
As a more critical issue/question I have HP Aruba switches 5406rzl and HP 2910 and HP 2920's I've been running into issues where the mac addresses don't show up for most ports but then do for 3-4, and for a few the IP matches up for most they don't. I'm starting to think that a new device scanning methods may be needed.

**Comment by jkw117** [comment]
I'll work up a word document with screenshots, and results of a few of the commands.

**Comment by netniV** [comment]
You can copy and paste images straight into the GitHub entry box.  It's far better than using a separate word document as the images appear straight in your post.  Please extract the images and paste separately.

**Comment by jkw117** [comment]
Date | Mon, 06 Aug 2018 13:37:26 -0400
-- | --
Cacti Version | 1.1.38
Cacti OS | unix
RSA Fingerprint | 8e:35:ed:b3:e0:56:ce:39:44:a3:27:53:2f:d6:3a:e1
NET-SNMP Version | NET-SNMP version: 5.7.3
RRDtool Version | RRDtool 1.7.x
Devices | 117
Graphs | 3512


The HP – Aruba switches I have are HP 2920, HP 2910, HP 5406 switches. 

php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=36 -d
2018/08/06 12:59:10 - MACTRACK HOST: 172.19.85.55 is alive, processing has begun.
2018/08/06 12:59:10 - MACTRACK DEBUG: Scanning function is get_procurve_ngi_switch_ports
2018/08/06 12:59:10 - MACTRACK DEBUG: VLAN data collected. There are 7 VLANS.
2018/08/06 12:59:10 - MACTRACK DEBUG: ifIndexes data collection complete
2018/08/06 12:59:10 - MACTRACK DEBUG: ifIndexes data collection complete. '40' rows found!
2018/08/06 12:59:10 - MACTRACK DEBUG: ifTypes data collection complete. '40' rows found!
2018/08/06 12:59:10 - MACTRACK DEBUG: ifNames data collection complete. '40' rows found!
2018/08/06 12:59:11 - MACTRACK DEBUG: ipAddrTable scanning for link ports data collection complete. '2' rows found!
2018/08/06 12:59:11 - MACTRACK DEBUG: ifSpeed data collection complete. '40' rows found!
2018/08/06 12:59:11 - MACTRACK DEBUG: ifHighSpeed data collection complete. '40' rows found!
2018/08/06 12:59:11 - MACTRACK DEBUG: ifDuplex data collection complete. '24' rows found!
2018/08/06 12:59:11 - MACTRACK DEBUG: ifDescr data collection complete. '40' rows found!
2018/08/06 12:59:11 - MACTRACK DEBUG: ifMtu data collection complete. '40' rows found!
2018/08/06 12:59:11 - MACTRACK DEBUG: ifPhysAddress data collection complete. '40' rows found!
2018/08/06 12:59:11 - MACTRACK DEBUG: ifAdminStatus data collection complete. '40' rows found!
2018/08/06 12:59:11 - MACTRACK DEBUG: ifOperStatus data collection complete. '40' rows found!
2018/08/06 12:59:11 - MACTRACK DEBUG: ifLastChange data collection complete. '40' rows found!
2018/08/06 12:59:11 - MACTRACK DEBUG: ifInOctets data collection complete. '40' rows found!
2018/08/06 12:59:11 - MACTRACK DEBUG: ifOutOctets data collection complete. '40' rows found!
2018/08/06 12:59:11 - MACTRACK DEBUG: ifHCInOctets data collection complete. '40' rows found!
2018/08/06 12:59:11 - MACTRACK DEBUG: ifHCOutOctets data collection complete. '40' rows found!
2018/08/06 12:59:11 - MACTRACK DEBUG: ifInMulticastPkts data collection complete. '40' rows found!
2018/08/06 12:59:11 - MACTRACK DEBUG: ifOutMulticastPkts data collection complete. '40' rows found!
2018/08/06 12:59:11 - MACTRACK DEBUG: ifInBroadcastPkts data collection complete. '40' rows found!
2018/08/06 12:59:11 - MACTRACK DEBUG: ifOutBroadcastPkts data collection complete. '40' rows found!
2018/08/06 12:59:11 - MACTRACK DEBUG: ifInUcastPkts data collection complete. '40' rows found!
2018/08/06 12:59:11 - MACTRACK DEBUG: ifOutUcastPkts data collection complete. '40' rows found!
2018/08/06 12:59:11 - MACTRACK DEBUG: ifInDiscards data collection complete. '40' rows found!
2018/08/06 12:59:11 - MACTRACK DEBUG: ifInErrors data collection complete. '40' rows found!
2018/08/06 12:59:11 - MACTRACK DEBUG: ifInUnknownProtos data collection complete. '40' rows found!
2018/08/06 12:59:11 - MACTRACK DEBUG: ifOutDiscards data collection complete. '40' rows found!
2018/08/06 12:59:11 - MACTRACK DEBUG: ifOutErrors data collection complete. '40' rows found!
2018/08/06 12:59:11 - MACTRACK ifInterfaces assembly complete: 20828
2018/08/06 12:59:11 - MACTRACK DEBUG: Adding IfInterfaces Records
2018/08/06 12:59:11 - MACTRACK DEBUG: ifInterfaces assembly complete.
2018/08/06 12:59:11 - MACTRACK DEBUG: Port number information collected.
2018/08/06 12:59:12 - MACTRACK INDEX: '0' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '1' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '2' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '3' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '4' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '5' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '6' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '7' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '8' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '9' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '10' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '11' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '12' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '13' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '14' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '15' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '16' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '17' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '18' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '19' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '20' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '21' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '22' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '23' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '24' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '25' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '26' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '27' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '28' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '29' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '30' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '31' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '32' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '33' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '34' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '35' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '36' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '37' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '38' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '39' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '40' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '41' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '42' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '43' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '44' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '45' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '46' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '47' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '48' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '49' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '50' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '51' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '52' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '53' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '54' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '55' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '56' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '57' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '58' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '59' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '60' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '61' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '62' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '63' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '64' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '65' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '66' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '67' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '68' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '69' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '70' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '71' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '72' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '73' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '74' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '75' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '76' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '77' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '78' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '79' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '80' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '81' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '82' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '83' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '84' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '85' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '86' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '87' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '88' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '89' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '90' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '91' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '92' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '93' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '94' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '95' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '96' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '97' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '98' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '99' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '100' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '101' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '102' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '103' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '104' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '105' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '106' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '107' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '108' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '109' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '110' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '111' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '112' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '113' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '114' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '115' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '116' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '117' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK INDEX: '118' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 12:59:12 - MACTRACK DEBUG: Port mac address information collected.
INFO: HOST: 172.19.85.55, TYPE: HP J9146A 2910al-24G-PoE Switch, revisio, TOTAL PORTS: 24, ACTIVE PORTS: 0
It ends up showing no macs (I’m assuming because it’s saying there’s no active ports.
![image](https://user-images.githubusercontent.com/12304270/43732905-eb1b1670-9980-11e8-9c9d-f250b1f55516.png)

 
php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=50 -d
2018/08/06 13:36:00 - MACTRACK HOST: 172.19.85.79 is alive, processing has begun.
2018/08/06 13:36:00 - MACTRACK DEBUG: Scanning function is get_procurve_ngi_switch_ports
2018/08/06 13:36:00 - MACTRACK DEBUG: VLAN data collected. There are 8 VLANS.
2018/08/06 13:36:00 - MACTRACK DEBUG: ifIndexes data collection complete
2018/08/06 13:36:00 - MACTRACK DEBUG: ifIndexes data collection complete. '67' rows found!
2018/08/06 13:36:01 - MACTRACK DEBUG: ifTypes data collection complete. '67' rows found!
2018/08/06 13:36:01 - MACTRACK DEBUG: ifNames data collection complete. '67' rows found!
2018/08/06 13:36:01 - MACTRACK DEBUG: ipAddrTable scanning for link ports data collection complete. '2' rows found!
2018/08/06 13:36:01 - MACTRACK DEBUG: ifSpeed data collection complete. '67' rows found!
2018/08/06 13:36:02 - MACTRACK DEBUG: ifHighSpeed data collection complete. '67' rows found!
2018/08/06 13:36:02 - MACTRACK DEBUG: ifDuplex data collection complete. '48' rows found!
2018/08/06 13:36:02 - MACTRACK DEBUG: ifDescr data collection complete. '67' rows found!
2018/08/06 13:36:02 - MACTRACK DEBUG: ifMtu data collection complete. '67' rows found!
2018/08/06 13:36:02 - MACTRACK DEBUG: ifPhysAddress data collection complete. '67' rows found!
2018/08/06 13:36:02 - MACTRACK DEBUG: ifAdminStatus data collection complete. '67' rows found!
2018/08/06 13:36:02 - MACTRACK DEBUG: ifOperStatus data collection complete. '67' rows found!
2018/08/06 13:36:02 - MACTRACK DEBUG: ifLastChange data collection complete. '67' rows found!
2018/08/06 13:36:02 - MACTRACK DEBUG: ifInOctets data collection complete. '67' rows found!
2018/08/06 13:36:02 - MACTRACK DEBUG: ifOutOctets data collection complete. '67' rows found!
2018/08/06 13:36:02 - MACTRACK DEBUG: ifHCInOctets data collection complete. '67' rows found!
2018/08/06 13:36:02 - MACTRACK DEBUG: ifHCOutOctets data collection complete. '67' rows found!
2018/08/06 13:36:02 - MACTRACK DEBUG: ifInMulticastPkts data collection complete. '67' rows found!
2018/08/06 13:36:02 - MACTRACK DEBUG: ifOutMulticastPkts data collection complete. '67' rows found!
2018/08/06 13:36:02 - MACTRACK DEBUG: ifInBroadcastPkts data collection complete. '67' rows found!
2018/08/06 13:36:02 - MACTRACK DEBUG: ifOutBroadcastPkts data collection complete. '67' rows found!
2018/08/06 13:36:02 - MACTRACK DEBUG: ifInUcastPkts data collection complete. '67' rows found!
2018/08/06 13:36:02 - MACTRACK DEBUG: ifOutUcastPkts data collection complete. '67' rows found!
2018/08/06 13:36:02 - MACTRACK DEBUG: ifInDiscards data collection complete. '67' rows found!
2018/08/06 13:36:02 - MACTRACK DEBUG: ifInErrors data collection complete. '67' rows found!
2018/08/06 13:36:03 - MACTRACK DEBUG: ifInUnknownProtos data collection complete. '67' rows found!
2018/08/06 13:36:03 - MACTRACK DEBUG: ifOutDiscards data collection complete. '67' rows found!
2018/08/06 13:36:03 - MACTRACK DEBUG: ifOutErrors data collection complete. '67' rows found!
2018/08/06 13:36:03 - MACTRACK ifInterfaces assembly complete: 31777
2018/08/06 13:36:03 - MACTRACK DEBUG: Adding IfInterfaces Records
2018/08/06 13:36:03 - MACTRACK DEBUG: ifInterfaces assembly complete.
2018/08/06 13:36:03 - MACTRACK DEBUG: Port number information collected.
2018/08/06 13:36:03 - MACTRACK INDEX: '0' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 13:36:03 - MACTRACK INDEX: '1' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 13:36:03 - MACTRACK INDEX: '2' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 13:36:03 - MACTRACK INDEX: '3' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 13:36:03 - MACTRACK INDEX: '4' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 13:36:03 - MACTRACK INDEX: '5' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 13:36:03 - MACTRACK INDEX: '6' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 13:36:03 - MACTRACK INDEX: '7' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 13:36:03 - MACTRACK INDEX: '8' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 13:36:03 - MACTRACK INDEX: '9' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 13:36:03 - MACTRACK INDEX: '10' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 13:36:03 - MACTRACK INDEX: '11' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 13:36:03 - MACTRACK INDEX: '12' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 13:36:03 - MACTRACK INDEX: '13' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 13:36:03 - MACTRACK INDEX: '14' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 13:36:03 - MACTRACK INDEX: '15' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 13:36:03 - MACTRACK INDEX: '16' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 13:36:03 - MACTRACK INDEX: '17' MAC ADDRESS: xx:xx:xx:xx:xx:xx:
2018/08/06 13:36:03 - MACTRACK DEBUG: Port mac address information collected.
2018/08/06 13:36:03 - MACTRACK VLAN: 233, NAME: Energy-Systems, PORT: 42, NAME: HVAC-NAE01-NAE5510, MAC: xx:xx:xx:xx:xx:xx:
2018/08/06 13:36:03 - MACTRACK VLAN: 26, NAME: Wireless-Management, PORT: 3, NAME: Courtroom 2C AP, MAC: xx:xx:xx:xx:xx:xx:
2018/08/06 13:36:03 - MACTRACK VLAN: 26, NAME: Wireless-Management, PORT: 1, NAME: Courtroom 2D AP, MAC: xx:xx:xx:xx:xx:xx:
INFO: HOST: 172.19.85.79, TYPE: HP J9729A 2920-48G-POE+ Switch, revision, TOTAL PORTS: 49, ACTIVE PORTS: 32018/08/06 13:36:03 - MACTRACK DEBUG: MAC Address 'xx:xx:xx:xx:xx:xx:' on device 'CH-2F-9' is NOT Authorized
2018/08/06 13:36:03 - MACTRACK DEBUG: MAC Address 'xx:xx:xx:xx:xx:xx:' on device 'CH-2F-9' is NOT Authorized
2018/08/06 13:36:03 - MACTRACK DEBUG: MAC Address 'xx:xx:xx:xx:xx:xx:' on device 'CH-2F-9' is NOT Authorized
This one shows a few mac's even though the switch in both cases is fully populated.


![image](https://user-images.githubusercontent.com/12304270/43732913-f568c6c2-9980-11e8-9804-dc313fe3af8b.png)


 

php -q poller_mactrack.php -f -d
2018/08/06 13:09:59 - MACTRACK DEBUG: About to enter Device Tracking poller processing
2018/08/06 13:09:59 - MACTRACK DEBUG: Into Processing.  Checking to determine if it's time to run.
2018/08/06 13:09:59 - MACTRACK DEBUG: Checking if user changed the start time
2018/08/06 13:09:59 - MACTRACK DEBUG: Checking if user changed the maintenance time
2018/08/06 13:09:59 - MACTRACK DEBUG: The next run time has been determined to be at '2018-08-06 13:17:16'
2018/08/06 13:09:59 - MACTRACK DEBUG: The next database maintenance run time has been determined to be at '2018-08-07 4:00:00'
2018/08/06 13:09:59 - MACTRACK DEBUG: Either a scan has been forced, or it's time to check for macs
2018/08/06 13:09:59 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=56 -d
2018/08/06 13:09:59 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=56 -d]
2018/08/06 13:09:59 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=41 -d
2018/08/06 13:09:59 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=41 -d]
2018/08/06 13:09:59 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=40 -d
2018/08/06 13:09:59 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=40 -d]
2018/08/06 13:09:59 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=39 -d
2018/08/06 13:09:59 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=39 -d]
2018/08/06 13:09:59 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=38 -d
2018/08/06 13:09:59 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=38 -d]
2018/08/06 13:09:59 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=57 -d
2018/08/06 13:09:59 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=57 -d]
2018/08/06 13:09:59 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=37 -d
2018/08/06 13:09:59 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=37 -d]
2018/08/06 13:09:59 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=36 -d
2018/08/06 13:09:59 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=36 -d]
2018/08/06 13:09:59 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=29 -d
2018/08/06 13:09:59 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=29 -d]
2018/08/06 13:09:59 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=28 -d
2018/08/06 13:09:59 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=28 -d]
2018/08/06 13:09:59 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=94 -d
2018/08/06 13:09:59 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=94 -d]
2018/08/06 13:09:59 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=93 -d
2018/08/06 13:09:59 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=93 -d]
2018/08/06 13:09:59 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=48 -d
2018/08/06 13:09:59 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=48 -d]
2018/08/06 13:09:59 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=42 -d
2018/08/06 13:09:59 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=42 -d]
2018/08/06 13:09:59 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=43 -d
2018/08/06 13:09:59 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=43 -d]
2018/08/06 13:09:59 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=44 -d
2018/08/06 13:09:59 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=44 -d]
2018/08/06 13:09:59 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=9 -d
2018/08/06 13:09:59 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=9 -d]
2018/08/06 13:09:59 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=55 -d
2018/08/06 13:09:59 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=55 -d]
2018/08/06 13:09:59 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=54 -d
2018/08/06 13:09:59 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=54 -d]
2018/08/06 13:09:59 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=53 -d
2018/08/06 13:09:59 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=53 -d]
2018/08/06 13:09:59 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=52 -d
2018/08/06 13:09:59 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=52 -d]
2018/08/06 13:09:59 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=51 -d
2018/08/06 13:09:59 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=51 -d]
2018/08/06 13:09:59 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=8 -d
2018/08/06 13:09:59 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=8 -d]
2018/08/06 13:09:59 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=50 -d
2018/08/06 13:09:59 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=50 -d]
2018/08/06 13:09:59 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=49 -d
2018/08/06 13:09:59 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=49 -d]
2018/08/06 13:09:59 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=47 -d
2018/08/06 13:09:59 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=47 -d]
2018/08/06 13:09:59 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=46 -d
2018/08/06 13:09:59 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=46 -d]
2018/08/06 13:09:59 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=45 -d
2018/08/06 13:09:59 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=45 -d]
2018/08/06 13:09:59 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=92 -d
2018/08/06 13:09:59 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=92 -d]
2018/08/06 13:09:59 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=74 -d
2018/08/06 13:09:59 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=74 -d]
2018/08/06 13:10:01 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_resolver.php -d]
2018/08/06 13:10:01 - MACTRACK DEBUG: DNS Resolver process launched
2018/08/06 13:10:01 - MACTRACK DEBUG: A process cycle launch just completed.
2018/08/06 13:10:03 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=66 -d
2018/08/06 13:10:03 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=66 -d]
2018/08/06 13:10:03 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=67 -d
2018/08/06 13:10:03 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=67 -d]
2018/08/06 13:10:03 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=68 -d
2018/08/06 13:10:03 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=68 -d]
2018/08/06 13:10:03 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=69 -d
2018/08/06 13:10:03 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=69 -d]
2018/08/06 13:10:03 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=70 -d
2018/08/06 13:10:03 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=70 -d]
2018/08/06 13:10:03 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=71 -d
2018/08/06 13:10:03 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=71 -d]
2018/08/06 13:10:03 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=72 -d
2018/08/06 13:10:03 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=72 -d]
2018/08/06 13:10:03 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=73 -d
2018/08/06 13:10:03 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=73 -d]
2018/08/06 13:10:04 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=65 -d
2018/08/06 13:10:04 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=65 -d]
2018/08/06 13:10:04 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=11 -d
2018/08/06 13:10:04 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=11 -d]
2018/08/06 13:10:04 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=59 -d
2018/08/06 13:10:04 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=59 -d]
2018/08/06 13:10:04 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=60 -d
2018/08/06 13:10:04 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=60 -d]
2018/08/06 13:10:04 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=61 -d
2018/08/06 13:10:04 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=61 -d]
2018/08/06 13:10:04 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=63 -d
2018/08/06 13:10:04 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=63 -d]
2018/08/06 13:10:04 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=62 -d
2018/08/06 13:10:04 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=62 -d]
2018/08/06 13:10:04 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=64 -d
2018/08/06 13:10:04 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=64 -d]
2018/08/06 13:10:04 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=1 -d
2018/08/06 13:10:04 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=1 -d]
2018/08/06 13:10:04 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=35 -d
2018/08/06 13:10:04 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=35 -d]
2018/08/06 13:10:04 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=34 -d
2018/08/06 13:10:04 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=34 -d]
2018/08/06 13:10:04 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=3 -d
2018/08/06 13:10:04 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=3 -d]
2018/08/06 13:10:04 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=31 -d
2018/08/06 13:10:04 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=31 -d]
2018/08/06 13:10:04 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=32 -d
2018/08/06 13:10:04 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=32 -d]
2018/08/06 13:10:04 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=33 -d
2018/08/06 13:10:04 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=33 -d]
2018/08/06 13:10:04 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=7 -d
2018/08/06 13:10:04 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=7 -d]
2018/08/06 13:10:04 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=14 -d
2018/08/06 13:10:04 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=14 -d]
2018/08/06 13:10:04 - MACTRACK DEBUG: A process cycle launch just completed.
2018/08/06 13:10:06 - MACTRACK DEBUG: Interum IP addresses to MAC addresses association pass complete.
2018/08/06 13:10:06 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=96 -d
2018/08/06 13:10:06 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=96 -d]
2018/08/06 13:10:06 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=13 -d
2018/08/06 13:10:06 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=13 -d]
2018/08/06 13:10:06 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=12 -d
2018/08/06 13:10:06 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=12 -d]
2018/08/06 13:10:06 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=16 -d
2018/08/06 13:10:06 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=16 -d]
2018/08/06 13:10:06 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=10 -d
2018/08/06 13:10:06 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=10 -d]
2018/08/06 13:10:06 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=30 -d
2018/08/06 13:10:06 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=30 -d]
2018/08/06 13:10:06 - MACTRACK DEBUG: A process cycle launch just completed.
2018/08/06 13:10:08 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=19 -d
2018/08/06 13:10:08 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=19 -d]
2018/08/06 13:10:08 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=17 -d
2018/08/06 13:10:08 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=17 -d]
2018/08/06 13:10:08 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=18 -d
2018/08/06 13:10:08 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=18 -d]
2018/08/06 13:10:08 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=15 -d
2018/08/06 13:10:08 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=15 -d]
2018/08/06 13:10:08 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=20 -d
2018/08/06 13:10:08 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=20 -d]
2018/08/06 13:10:08 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=2 -d
2018/08/06 13:10:08 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=2 -d]
2018/08/06 13:10:08 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=21 -d
2018/08/06 13:10:08 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=21 -d]
2018/08/06 13:10:08 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=22 -d
2018/08/06 13:10:08 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=22 -d]
2018/08/06 13:10:08 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=95 -d
2018/08/06 13:10:08 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=95 -d]
2018/08/06 13:10:08 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=23 -d
2018/08/06 13:10:08 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=23 -d]
2018/08/06 13:10:08 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=24 -d
2018/08/06 13:10:08 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=24 -d]
2018/08/06 13:10:08 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=25 -d
2018/08/06 13:10:08 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=25 -d]
2018/08/06 13:10:08 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=26 -d
2018/08/06 13:10:08 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=26 -d]
2018/08/06 13:10:08 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=27 -d
2018/08/06 13:10:08 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=27 -d]
2018/08/06 13:10:08 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=82 -d
2018/08/06 13:10:08 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=82 -d]
2018/08/06 13:10:08 - MACTRACK DEBUG: A process cycle launch just completed.
2018/08/06 13:10:10 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=81 -d
2018/08/06 13:10:10 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=81 -d]
2018/08/06 13:10:10 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=80 -d
2018/08/06 13:10:10 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=80 -d]
2018/08/06 13:10:10 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=85 -d
2018/08/06 13:10:10 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=85 -d]
2018/08/06 13:10:10 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=84 -d
2018/08/06 13:10:10 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=84 -d]
2018/08/06 13:10:10 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=83 -d
2018/08/06 13:10:10 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=83 -d]
2018/08/06 13:10:10 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=75 -d
2018/08/06 13:10:10 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=75 -d]
2018/08/06 13:10:10 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=76 -d
2018/08/06 13:10:10 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=76 -d]
2018/08/06 13:10:10 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=77 -d
2018/08/06 13:10:10 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=77 -d]
2018/08/06 13:10:10 - MACTRACK DEBUG: A process cycle launch just completed.
2018/08/06 13:10:12 - MACTRACK DEBUG: Interum IP addresses to MAC addresses association pass complete.
2018/08/06 13:10:12 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=78 -d
2018/08/06 13:10:12 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=78 -d]
2018/08/06 13:10:12 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=79 -d
2018/08/06 13:10:12 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=79 -d]
2018/08/06 13:10:12 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=91 -d
2018/08/06 13:10:12 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=91 -d]
2018/08/06 13:10:12 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=90 -d
2018/08/06 13:10:12 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=90 -d]
2018/08/06 13:10:12 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=58 -d
2018/08/06 13:10:12 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=58 -d]
2018/08/06 13:10:12 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=87 -d
2018/08/06 13:10:12 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=87 -d]
2018/08/06 13:10:12 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=86 -d
2018/08/06 13:10:12 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=86 -d]
2018/08/06 13:10:12 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=88 -d
2018/08/06 13:10:12 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=88 -d]
2018/08/06 13:10:12 - MACTRACK CMD: /usr/bin/php -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=89 -d
2018/08/06 13:10:12 - POLLER: Poller[1] DEBUG: About to Spawn a Remote Process [CMD: /usr/bin/php, ARGS:  -q /usr/share/cacti/site/plugins/mactrack/mactrack_scanner.php -id=89 -d]
2018/08/06 13:10:12 - MACTRACK DEBUG: A process cycle launch just completed.
2018/08/06 13:10:16 - MACTRACK DEBUG: Waiting on 13 with id = [-1, 0, 1, 2, 10, 58, 63, 78, 79, 86, 87, 88, 89, 91, 95] to complete prior to exiting.
2018/08/06 13:10:18 - MACTRACK DEBUG: Interum IP addresses to MAC addresses association pass complete.
2018/08/06 13:10:18 - MACTRACK DEBUG: Waiting on 4 with id = [-1, 0, 1, 2, 10, 95] to complete prior to exiting.
2018/08/06 13:10:20 - MACTRACK DEBUG: Interum IP addresses to MAC addresses association pass complete.
2018/08/06 13:10:20 - MACTRACK DEBUG: Waiting on 3 with id = [-1, 0, 1, 2, 95] to complete prior to exiting.
2018/08/06 13:10:22 - MACTRACK DEBUG: Interum IP addresses to MAC addresses association pass complete.
2018/08/06 13:10:22 - MACTRACK DEBUG: Waiting on 2 with id = [-1, 0, 1, 2] to complete prior to exiting.
2018/08/06 13:10:24 - MACTRACK DEBUG: Interum IP addresses to MAC addresses association pass complete.
2018/08/06 13:10:24 - MACTRACK DEBUG: Waiting on 1 with id = [-1, 0, 1] to complete prior to exiting.
2018/08/06 13:10:27 - MACTRACK DEBUG: Interum IP addresses to MAC addresses association pass complete.
2018/08/06 13:10:27 - MACTRACK DEBUG: Waiting on 1 with id = [-1, 0, 1] to complete prior to exiting.
2018/08/06 13:10:29 - MACTRACK DEBUG: Interum IP addresses to MAC addresses association pass complete.
2018/08/06 13:10:29 - MACTRACK DEBUG: Waiting on 1 with id = [-1, 0, 1] to complete prior to exiting.
2018/08/06 13:10:31 - MACTRACK DEBUG: Interum IP addresses to MAC addresses association pass complete.
2018/08/06 13:10:31 - MACTRACK DEBUG: Waiting on 1 with id = [-1, 0, 1] to complete prior to exiting.
2018/08/06 13:10:33 - MACTRACK DEBUG: Interum IP addresses to MAC addresses association pass complete.
2018/08/06 13:10:33 - MACTRACK DEBUG: Waiting on 1 with id = [-1, 0, 1] to complete prior to exiting.
2018/08/06 13:10:35 - MACTRACK DEBUG: Interum IP addresses to MAC addresses association pass complete.
2018/08/06 13:10:35 - MACTRACK DEBUG: Waiting on 0 with id = [-1, 0] to complete prior to exiting.
2018/08/06 13:10:35 - MACTRACK DEBUG: Interum IP addresses to MAC addresses association pass complete.
2018/08/06 13:10:35 - MACTRACK DEBUG: MAC addresses to Vendor MACS association pass complete.
2018/08/06 13:10:35 - MACTRACK DEBUG: MAC VLANs in VLAN Table Updated.
2018/08/06 13:10:35 - MACTRACK DEBUG: Finished transferring scan results to main table.
2018/08/06 13:10:35 - MACTRACK DEBUG: Finished updating site table with collection statistics.
2018/08/06 13:10:36 - SYSTEM MACTRACK STATS: Time:37.0937 ConcurrentProcesses:30 Devices:0

Let me know what you'd like me to try, and or change at the moment I have set them to:
![image](https://user-images.githubusercontent.com/12304270/43733697-846e6208-9983-11e8-96ba-e557a7828066.png)

Basically tried all of them and this seems to present some results.

**Comment by cigamit** [comment]
I'm going to close.  However, there have been a number of changes recently.  Please update to the latest version, and then if this problem continues, you can open a new issue.  Sorry about that.
